src/components/sequencing/SequenceEditor.svelte

```diff
@@ -44,6 +45,7 @@
     const { typescript } = languages;
     const { typescriptDefaults } = typescript;
     const options = typescriptDefaults.getCompilerOptions();
+    editor = monaco.editor.create(document.createElement('div'));
```

Comment:
Can we somehow bind to the `editor` within the `ManacoEditor` component instead of creating a new one?